boltons/iterutils.py

```diff
@@ -374,6 +374,30 @@ def chunked_iter(src, size, **kw):
     return
 
 
+def chunked_filter(iterable, predicate, chunk_size):
+    """A version of :func:`filter` which will call predicate with a chunk of the iterable.
```

Comment:
I think this is a great description, though it's a good idea to style arguments when referring to them (italicize or monospace).